[DOCS] Update Use Great Expectations with Amazon Web Services using Athena (#8406)
great-expectations/great_expectations: docs/docusaurus/docs/guides/connecting_to_your_data/database/components/_datasource_athena_test.md

> Run the following command to prepare an empty Expectation suite:

Reviewer: Add the snippet referenced below here, feel free to update the text
````suggestion
Run the following code to connect to Athena, add an asset for your table, and build a batch request for your table:

```python name="tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py Connect and Build Batch Request"

```

Run the following code to prepare an empty Expectation suite:

````

> Run the following command to create a Batch Request:

Author: ```suggestion
Run the following code to create a Batch Request:
```

```python name="tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py Batch Request"

```

> Run the following command to prepare an empty Expectation suite:

Author: ```suggestion
Run the following code to prepare an empty Expectation suite:
```

```python name="tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py Create Expectation Suite"

```

> Run the following command to load data into a `Validator`:

Author: ```suggestion
Run the following code to load data into a `Validator`:
```

```python name="tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py Test Datasource with Validator"

```

> If the command executes successfully, it verifies that your Datasource is working.

Author: ```suggestion
When the code executes successfully, it indicates that your Data Source is working.
```
